src/main.js

```diff
@@ -55,8 +55,11 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
      * navigator.language in ExtensionLoader (when making require contexts for each
      * extension).
      */
+    if(window.location.search) {
+        var locale = /locale=([^&]+)&?/.exec(window.location.search)[1] || "en";
```

Comment:
This seems brittle. If the `if` bock doesn't evaluate as true then `locale` will be undefined below. I'd do something like:

```js
var locale = window.localStorage.getItem("locale") || (typeof (brackets) !== "undefined" ? brackets.app.language : navigator.language);

if (window.location.search) {
  locale = ...;
}
```